visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -44,12 +45,25 @@ export class CodeMapRenderService {
 	}
 
 	private getSortedNodes(map: CodeMapNode): Node[] {
-		const nodes: Node[] = TreeMapGenerator.createTreemapNodes(
-			map,
-			this.storeService.getState(),
-			this.metricService.getMetricData(),
-			this.storeService.getState().files.isDeltaState()
-		)
+		let nodes: Node[] = []
+		const state = this.storeService.getState()
+		const metricService = this.metricService.getMetricData()
```

Comment:
Rename to `metricData`, as you are just passing the data object, but not the service itself (: